From 564d39a5cb535a5c8301e6adf9f545985b7af10d Mon Sep 17 00:00:00 2001
From: Joseph Fernandes <josferna@redhat.com>
Date: Mon, 9 Nov 2015 20:36:11 +0530
Subject: [PATCH 099/131] tier/ctr: ignoring bitrot scrubber fops

We had missed adding  GF_CLIENT_PID_SCRUB to the internal fops
list of bitrot. Doing that in this fix

Backport of http://review.gluster.org/#/c/12555/

> Change-Id: I0c7de37e2bf625fb577a32a599a885ee95f5d3bd
> BUG: 1278326
> Signed-off-by: Joseph Fernandes <josferna@redhat.com>
> Reviewed-on: http://review.gluster.org/12555
> Tested-by: Gluster Build System <jenkins@build.gluster.com>
> Reviewed-by: Venky Shankar <vshankar@redhat.com>
> Reviewed-by: Dan Lambright <dlambrig@redhat.com>
> Tested-by: Dan Lambright <dlambrig@redhat.com>
Signed-off-by: Joseph Fernandes <josferna@redhat.com>

Change-Id: I526fef4659e81708743cb5d5b046f480371908ee
BUG: 1277126
Reviewed-on: https://code.engineering.redhat.com/gerrit/61241
Reviewed-by: Dan Lambright <dlambrig@redhat.com>
Tested-by: Dan Lambright <dlambrig@redhat.com>
---
 .../features/changetimerecorder/src/ctr-helper.h   |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/xlators/features/changetimerecorder/src/ctr-helper.h b/xlators/features/changetimerecorder/src/ctr-helper.h
index e833c87..bee1432 100644
--- a/xlators/features/changetimerecorder/src/ctr-helper.h
+++ b/xlators/features/changetimerecorder/src/ctr-helper.h
@@ -260,7 +260,8 @@ do {\
  * If a bitrot fop
  * */
 #define BITROT_FOP(frame)\
-        (frame->root->pid == GF_CLIENT_PID_BITD)
+        (frame->root->pid == GF_CLIENT_PID_BITD ||\
+        frame->root->pid == GF_CLIENT_PID_SCRUB)
 
 
 /*
-- 
1.7.1

